versiebeheer.md

```diff
@@ -0,0 +1,155 @@
+# Versie beheer en release management
+
+## API versies
+
+Versiebeheer is gebaseerd op [Semantic Versioning](https://semver.org) en de
+[API strategie](https://aandeslagmetdeomgevingswet.nl/digitaal-stelsel/technisch-aansluiten/standaarden/api-uri-strategie/).
+Elke  API heeft een eigen versienummer, opgebouwd uit `MAJOR.MINOR.PATCH` 
+element. Bijvoorbeeld versie `2.1.8`:
+
+Gegeven dit versienummer, elke ophoging van de:
+
+* `MAJOR` versie betreft een *backwards-incompatible* API wijziging,
+* `MINOR` versie betreft het toevoegen van een *backwards-compatible* API 
+  wijziging,
+* `PATCH` versie betreft het oplossen van *backwards-compatible* issues.
+
+Verder wordt een aantal labels gebruikt, achter het versienummer:
+
+* Als er geen aanduiding bij het versienummer staat, gaat het altijd om de
+  meest actuele productie versie van de API.
+* `supported` versies zijn oudere maar ondersteunde versies van de API,
+* `archived` versies zijn oudere, niet meer ondersteunde versies van de API.
+
+Voor meer interne doeleinden kennen we tenslotte nog de volgende labels achter
+het versienummer:
+
+* `RC` (Release Candidate) als de API versie feature-complete is. Alleen
+  kritieke issues worden in deze versie nog verwerkt,
+* `beta` als de versie gereed is voor testers en er geen grote wijzigingen meer
+  verwacht worden. Alleen essentiele zaken worden in dit stadium nog 
+  toegevoegd,
+* `alpha` als de versie nog in ontwikkeling is en er geen enkele garantie is
+  over de werking, stabiliteit e.d.
+
+### Voorbeelden van backwards-incompatible wijzigingen
+
+* Wijzigingen in de URL of fundamentale wijzigingen in de request/response
+  schema's die bij een resource horen,
+* Er is een data migratie nodig om van de vorige naar de nieuwe versie te komen.
+  Bijvoorbeeld omdat een attribuut wordt opgesplitst in meerdere attributen, of
+  juist worden samengevoegd,
+* Verwijderen, hernoemen of type-wijziging van een attribuut, query parameter
+  of header,
+* Verwijderen of hernoemen van een resource of API als geheel,
+* Toevoegen van een verplicht attribuut of header.
+
+**LET OP: We spreken altijd over backwards-incompatible wijzigingen vanuit het 
+provider perspectief! Een provider met API op versie `2.1.8` kan altijd goed
+communiceren met een consumer die `2.0.0` t/m `2.1.8` praat (kortgezegd,
+`2.x`) maar het is niet persé waar dat een consumer op versie `2.2.0` goed kan 
+communiceren met een API op versie `2.1.8`.**
+
+### Voorbeelden van backwards-compatible wijzigingen:
+
+* Toevoegen van attributen die optioneel zijn, `null` mogen zijn of een andere
+  standaard waarde hebben.
+* Toevoegen van een element aan een enumeratie,
```

Comment:
```suggestion
* Toevoegen van een element aan een enumeratie,
```
* Toevoegen van een element aan een enumeratie, 
_(we twijfelen er nog aan of dit wel een voorbeeld is  van een backwards-compatible wijziging. Daar moeten we dus nog naar kijken)_